feature-group-definitions/server-timing.yml

```
@@ -1,4 +1,5 @@
 name: Server timing
+description: Server timing is part of the Performance API and allows to annotate network requests with server timing information.
```

How would we name https://w3c.github.io/resource-timing/#dom-performanceresourcetiming if it was added as a feature? I think we should use that name here, and later we might even be able to cross-link features.

I think "performance resource timing API" and "`PerformanceResourceTiming` API" would both work. Picking one:

```suggestion
description: The `serverTiming` property of the `PerformanceResourceTiming` API contains server timing information about network requests.
```

---

@captainbrosset did you mean to leave this suggestion out?